docs/api/LEAGUE.md

```diff
@@ -65,3 +65,11 @@
     * Possible Values:
       * Season format: `yyyyyyyy`
         * Example: `20172018`
+
+  * `standingsType`
+    * Possible Values:
```

Comment:
Can you also include the other possible [types](https://statsapi.web.nhl.com/api/v1/standingsTypes) here?

Reply:
Done.